python/tests/stress/arcticdb/version_store/test_stress_finalize_stage_data.py

```diff
@@ -0,0 +1,111 @@
+"""
+Copyright 2024 Man Group Operations Limited
+
+Use of this software is governed by the Business Source License 1.1 included in the file licenses/BSL.txt.
+
+As of the Change Date specified in that file, in accordance with the Business Source License, use of this software will be governed by the Apache License, version 2.0.
+"""
+
+import copy
+import gc
+import time
+import numpy as np
+import pandas as pd
+from typing import Union, List
+from arcticdb.version_store.library import Library, StagedDataFinalizeMethod
+from arcticdb.config import Defaults, set_log_level
+from arcticdb.util.utils import CachedDFGenerator, TimestampNumber, stage_chunks
+
+from typing import Union
+import numpy as np
+import pandas as pd
+
+# Uncomment for logging
+# set_log_level(default_level="DEBUG", console_output=False, file_output_path="/tmp/arcticdb.log")
+
+def generate_chunk_sizes(number_chunks:np.uint32, min_rows:np.uint32=100, max_rows:np.uint32=10000) -> List[np.uint32]:
+    return np.random.randint(min_rows, max_rows, number_chunks, dtype=np.uint32)
+
+def test_finalize_monotonic_unique_chunks(arctic_library_lmdb):
+
+    options = [
+        {"chunks_descending" : False, "finalization_mode" : StagedDataFinalizeMethod.APPEND},
+        {"chunks_descending" : True, "finalization_mode" : StagedDataFinalizeMethod.APPEND},
+        {"chunks_descending" : False, "finalization_mode" : StagedDataFinalizeMethod.WRITE},
+        {"chunks_descending" : True, "finalization_mode" : StagedDataFinalizeMethod.WRITE},
+        ]
+
+    # Will hold the results after each iteration:
```

Comment:
Defining a small `dataclass` for the objects you put in to results will help to make this clearer without the need for the comment

Reply:
Will do that! thanks!